src/actions/index.js

```diff
@@ -148,7 +143,7 @@ export const createFolder = name => {
     try {
       folder = await cozy.client.files.createDirectory({
         name: name,
-        dirID: getState().folder.id
+        dirID: getState().view.displayedFolder.id
```

Comment:
yes, there still are places to refactor in order to get rid of these `getState()` antipatterns